### DISCUSSION TITLE
WSA 2306 Update!

### DISCUSSION THREAD

**Discussion by hamza-usmani** [microsoft/WSA#359] [Announcements]
# What's New

We've shipped an update for Windows Subsystem for Android™ on Windows 11 to the [Windows Subsystem for Android™ Preview Program](https://learn.microsoft.com/en-us/windows/android/wsa/preview-program). This update (2306.40000.1.0) includes improvements to advanced settings, fixes for OneDrive folders, and several requested features.

## Amazon Appstore on Windows 11 is now generally available

Learn more about it: [**General Availability Announcement**](https://developer.amazon.com/apps-and-games/blogs/2023/07/appstore-on-windows-11-generally-available). Get started with the [Amazon Developer Portal](https://developer.amazon.com/apps-and-games/appstore-on-windows-11).

## Release Notes

- Camera compatibility improvements
- Fullscreen mode with F11 now displays hover taskbar to improve mouse and touch experience
- Local networking (under Advanced settings - Experimental features) now replacing "Advanced networking". Android apps can connect to devices on the same network, on all versions of Windows, respect Windows firewall rules, and work with VPNs
- Share user folders (under Advanced settings - Experimental features) now gives users the option to change their default shared folder
- The subsystem will now prompt if an app is trying to use a permission that the subsystem does not have
- The subsystem has switched from EROFS to EXT4 for read-only disks
- Fix for OneDrive folders not showing up in Android apps
- Support for drag and drop for more file types (feature request #335)
- Improvements to picture-in-picture (new UI buttons when in PIP)
- Partially running mode now enabled by default for devices with at least 16 GB of memory
- Stability fixes for ARM devices
- Linux kernel updated to 5.15.104
- Android 13 security updates

# Giving feedback

If you are having issues with Windows Subsystem for Android™ – please file feedback via Feedback Hub under Apps > Windows Subsystem for Android™. For more information about troubleshooting and submitting feedback: [Troubleshooting and FAQ for mobile apps on Windows](https://support.microsoft.com/en-us/windows/troubleshooting-and-faq-for-mobile-apps-on-windows-d6062afd-98a1-4018-a7c8-6b3b680a2ea5).

Thanks,
Windows Subsystem for Android™ Team

**Comment by HotCakeX** [comment]
Thank you, the file sharing specially with OneDrive works perfectly now!

Although i have one concern. I use Controlled Folder Access in Microsoft Defender and have added my entire OneDrive folder to it. I noticed that I could delete any files using WSA from my protected folders and it wasn't stopping me. The files wouldn't go to the recycle bin either, just completely deleted. 

So I assume this is an expected behavior and WSA is added to the [list of trusted apps by Controlled Folder Access](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/controlled-folders?view=o365-worldwide#how-does-controlled-folder-access-work)?

**Comment by HotCakeX** [comment]
In the road map it says "Local network access by default" for the future update but whenever I turn on advanced networking, all my Android apps lose Internet connectivity. I've used multiple VPN clients and advanced networking has never worked with any of them. I think if it's going to be turned on by default in a future update, its problem with VPNs should be solved.

A lot of VPNs also have features such as kill switch or lock down mode, they prevent any connection when the VPN suddenly disconnects or when it's not connected. I also find these features to be incompatible with the WSA advanced networking.

**Comment by ZzZombo** [comment]
Please tell us more about the camera improvements. Thanks in advance!

**Comment by omidsky2600** [comment]
Support keyboard windows devices with touchscreen ?
Don't Support keyboard touchscreen !

**Comment by omidsky2600** [comment]
why doesn't it show the keyboard when l installed the keyboard app ?
I installed. The. Google keyboard. But it does. Not. Show the letters
Every keyboard program I install is the same
There was no such problem before when there was Android 12
This happened when Android 13 was released

**Comment by AJolly** [comment]
Local network access does not seem to work for me - it gives me an IP on the local LAN, but the only device it can talk to is the network gateway router.  

With local network access turned off, it gives me an IP on a different subnet, but I can talk to devices on the local PC LAN.  

Tested with various network scanners / Amazon FireTV remote.

**Comment by malikamir2647** [comment]
Kindly guide us about local network access because it is not working 😕

**Comment by omidsky2600** [comment]
Any keyboard didn't work, microsoft swift keyboard, yandex keyboard, anyone didn't work
gboard is not showing up. gboard is working, i am able to type blindly. but the keyboard is not visible. if i tap somewhere in the keyboard area, i am able to type thats it. its not visible. can someone help with it

**Comment by kpx1894** [comment]
This update has broken network access. I tried every possible way so far to disable automatic updates but every morning I am getting back this new version. I have to uninstall it, reinstall old version, reinstall Android app that I am using, configure everything from scratch. Anyone knows any trick that would prevent WSA from updating automatically?
